examples/prompty/basic/prompty-quickstart.ipynb

```diff
@@ -79,10 +79,79 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from promptflow.core import Flow\n",
+    "from promptflow.core import Prompty, AzureOpenAIModelConfiguration, OpenAIModelConfiguration\n",
     "\n",
     "# load prompty as a flow\n",
-    "f = Flow.load(\"basic.prompty\")\n",
+    "f = Prompty.load(source=\"basic.prompty\")\n",
+    "\n",
+    "# override model configuration of prompty with dict\n",
+    "# override_model = {\n",
+    "#     \"configuration\": {\n",
+    "#         \"type\": \"azure_openai\",\n",
+    "#         \"azure_deployment\": \"gpt-35-turbo\",\n",
+    "#         \"api_key\": \"${env:AZURE_OPENAI_API_KEY}\",\n",
+    "#         \"azure_endpoint\": \"${env:AZURE_OPENAI_ENDPOINT}\",\n",
+    "#     },\n",
+    "#     \"parameters\": {\n",
+    "#         \"max_token\": 512\n",
+    "#     }\n",
+    "# }\n",
+    "# f = Prompty.load(source=\"basic.prompty\", model=override_model)\n",
+    "\n",
+    "# override configuration with AzureOpenAIModelConfiguration\n",
+    "# configuration = AzureOpenAIModelConfiguration(\n",
+    "#     azure_endpoint=\"${env:AZURE_OPENAI_ENDPOINT}\",\n",
+    "#     api_key=\"${env:AZURE_OPENAI_API_KEY}\",\n",
+    "#     azure_deployment=\"gpt-35-turbo\"\n",
+    "# )\n",
+    "# override_model = {\n",
+    "#     \"configuration\": configuration,\n",
+    "#     \"parameters\": {\n",
+    "#         \"max_token\": 512\n",
+    "#     }\n",
+    "# }\n",
+    "# f = Prompty.load(source=\"basic.prompty\", model=override_model)\n",
+    "\n",
+    "# override configuration with created connection in AzureOpenAIModelConfiguration\n",
+    "# configuration = AzureOpenAIModelConfiguration(\n",
```

Comment:
put the conneciton one into the chat notebook which is using connection?

Reply:
move connection into chat notebook